### PR TITLE
feat(1493): events have timezones

### PR DESCRIPTION
**Pull request by jhackett1**
fixes: #1493

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -235,6 +235,8 @@
     t.boolean  "confirmation_required", default: false
     t.boolean  "surveys_required",      default: false
     t.string   "audience"
+    t.boolean  "remote",                default: false, null: false
```
Comment (matyikriszta) [inline db/schema.rb:238]: @jhackett1 this change sneaked in here from another P.

**Review by KimberleyCook — APPROVED** [review]
🎉

**Comment by biggianteye** [comment]
This PR has been approved but hasn't been merged. Is there anything that's missing that would stop it from being merged (other than being out of sync given how old it is)?

**Comment by matyikriszta** [comment]
@biggianteye yes, we have tested this PR further and it does not actually work. I'm going to remove the Approval for now. We'll have to revisit this issue, it's on our roadmap.

File: db/schema.rb [hunk 2]
```diff
@@ -224,6 +224,7 @@
     t.boolean "surveys_required", default: false
     t.string "audience"
     t.boolean "virtual", default: false, null: false
+    t.string "timezone", default: "London", null: false
```
Comment (matyikriszta) [inline db/schema.rb:227]: Note to self, time zone is two words so change this to time_zone

**Review by gnclmorais — APPROVED** [review]
Nice change! Seems to work locally:
image

**Review by matyikriszta — APPROVED** [review]

**Pull request merged**